src/queries.ts

```diff
@@ -81,8 +81,8 @@ export const runQueries = async <T>(
 
   const startFormatting = performance.now();
 
-  for (let i = 0; i < results.length; i++) {
-    const result = results[i];
+  for (let index = 0; index < results.length; index++) {
+    const result = results[index];
```

Comment:
```suggestion
  for (const result of results) {
```

I'll let you decide if that makes more sense, considering there are some reassignments being made below. I personally prefer this more modern syntax!

Reply:
I used it because `for` loops with indexes are generally [faster](https://dev.to/siddiqus/which-for-loop-is-the-fastest-in-javascript-4hdf) than `for` loops with `of`, and since performance matters a lot here (we're inside a database client), I chose that one despite the syntax being longer.